app/containers/Header/styledComponents/searchForm.js

```diff
@@ -0,0 +1,7 @@
+import styled from 'styled-components';
```

Comment:
Do not create subfolder for styled components
file name needs to start from capital letter